src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    public void handleOperation(FruitTransaction fruitTransaction, Storage storage) {
```

Comment:
The `handleOperation` method should also return the updated `Storage` object or update it in place. If the intention is to update the `Storage` in place, then the method signature doesn't need to change, but if a new `Storage` should be returned, the method signature should reflect that.